### DISCUSSION TITLE
Manage API Keys

### DISCUSSION THREAD

**Discussion by tapter-mwm** [ovh/manager#7797] [Q&A]
Hi, 

I've created an API key through this url `https://www.ovh.com/auth/api/createToken`, but i can't seem to find any place on the ovh cloud console to manage existing api keys. Is there such thing on the console ? 

How are you suppose to list & manage your keys otherwise ?

Thanks for the help !

**Comment by NoArmyKnife** [comment]
Had a hard time to find the info, thought I should share it :
https://gandrille.github.io/linux-notes/Web_API/OVH_API/OVH_API_Keys_management.html

**Reply by David-** [reply]
New URL : [https://gandrille.github.io/linux-notes/Web-API/OVH-API/OVH-API-Keys-management/OVH-API-Keys-management](https://gandrille.github.io/linux-notes/Web-API/OVH-API/OVH-API-Keys-management/OVH-API-Keys-management)

**Comment by MaesterZ** [comment]
Simple Go CLI I forked https://gitlab.com/MaesterZ/ovh-akm
